### DISCUSSION TITLE
Submit api not working

### DISCUSSION THREAD

**Discussion by mukeshpwc0905** [argoproj/argo-workflows#9113] [Q&A] [answered]
Hey all!

Below is the my WorkflowTemplate which already submitted argo server. now i want to use api/v1/workflows/argo/submit api to send parameters.

apiVersion: argoproj.io/v1alpha1
kind: WorkflowTemplate
metadata:
  name: parameter-task
spec:
  entrypoint: main
  arguments:
    parameters:
    - name: workflow-param-1
      value: "Mukesh Shah"
    - name: workflow-param-2
      value: "Rohit Gurjar"
  templates:
  - name: main
    dag:
      tasks:
      - name: step-a 
        template: step-template-a
        arguments:
          parameters:
          - name: template-param-1
            value: "{{workflow.parameters.workflow-param-1}}"
          - name: template-param-2
            value: "{{workflow.parameters.workflow-param-2}}"

  - name: step-template-a
    inputs:
      parameters:
        - name: template-param-1
        - name: template-param-2
    script:
      image: alpine
      command: [/bin/sh]
      source: |
          echo "{{inputs.parameters.template-param-1}} {{inputs.parameters.template-param-2}}"

below one the request :

{
  "resourceKind": "WorkflowTemplate",
  "resourceName": "parameter-task",
  "submitOptions": {
    "parameters": ["workflow-param-1=ABC", "workflow-param-2=XYZ"]
  }
}

When I am trying to submit this request it was giving me error like : 
{
    "code": 5,
    "message": "workflowtemplates.argoproj.io \"parameter-task\" not found"
}

**Comment by terrytangyuan** [comment, marked as answer]
Your workflow template `parameter-task` does not exist yet.
